tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+from snuba.environment import setup_sentry
+
+
+def pytest_configure():
```

Comment:
If I get this right this would be a fixture to be explicitly used in a test, so here it does nothing.
How would that work if the DSN environment variable is not set ?

Reply:
It is automatically invoked for all test runs (once per pytest invocation, not once per test)

If the DSN is empty the Sentry SDK executes (and therefore triggers the bug) but does not send the results anywhere.

Reply:
ok. thanks for the explanation.